WordPressKit/ActivityServiceRemote.swift

```diff
@@ -37,28 +37,31 @@ open class ActivityServiceRemote: ServiceRemoteWordPressComREST {
                                    group: [String] = [],
                                    success: @escaping (_ activities: [Activity], _ hasMore: Bool) -> Void,
                                    failure: @escaping (Error) -> Void) {
-        let endpoint = "sites/\(siteID)/activity"
-        let path = self.path(forEndpoint: endpoint, withVersion: ._2_0)
-        let pageNumber = (offset / count + 1)
-        var parameters: [String: AnyObject] = [
-            "number": count as AnyObject,
-            "page": pageNumber as AnyObject
-        ]
 
-        if !group.isEmpty {
-            parameters["group[]"] = group.joined(separator: ",") as AnyObject
-        }
+        var path = URLComponents(string: "sites/\(siteID)/activity")
+
+        path?.queryItems = group.map { URLQueryItem(name: "group[]", value: $0) }
+
+        let pageNumber = (offset / count + 1)
```

Comment:
I was a bit confused about the intention of the calculation, what about shifting the parentheses to make it a bit more clear?

```suggestion
        let pageNumber = (offset / count) + 1
```

Reply:
Done in f3bda1433353ce9f59a01f942d5ce0eea6e7e881